authentication/methods.mdx

```diff
@@ -261,5 +262,21 @@ func getClientToken(ctx context.Context) (*Response, error) {
 
 </CodeGroup>
 
-The client token found in the body of the response can be used in the authentication bearer header, as outlined in [Using Client Tokens](/authentication/using-tokens).
+The client token found in the body of the response can then be used to authenticate with Flipt as outlined in [Using Client Tokens](/authentication/using-tokens).
+
 The expiration can be used to schedule when to next request a new client token.
+
+## JSON Web Tokens
+
+[JSON Web Tokens](https://jwt.io/) (JWT) are an open, industry standard RFC 7519 method for representing claims securely between two parties. Flipt supports the use of externally created and signed JWTs as a method of authentication.
+
+JWT authentication is useful for scenarios where you want to integrate Flipt with an existing authentication system.
```

Comment:
Perhaps we should make it clear that another authentication system is not a requirement, you just need to generate a key pair and share them between service and Flipt.
```suggestion
JWT authentication is useful for scenarios where you want to integrate Flipt with an existing authentication system, or where you want to perform service to Flipt authentication without handling static client tokens.
```